pkgs/by-name/ud/udev-block-notify/package.nix

```
@@ -0,0 +1,58 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  multimarkdown,
+  coreutils,
+  libnotify,
+  udev,
+  systemdLibs,
+  glib,
+  pkg-config,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "udev-block-notify";
+  version = "0.7.11";
+
+  src = fetchFromGitHub {
+    owner = "eworm-de";
+    repo = "udev-block-notify";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-A0uhfb2mEAAJgxRkv+MWTk/9oFiz3r7deAlu1Kpk+CI=";
+  };
+
+  nativeBuildInputs = [
+    multimarkdown
+    coreutils
+    pkg-config
+  ];
+  buildInputs = [
+    libnotify
+    udev
+    systemdLibs
+    glib
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D -m0755 udev-block-notify $out/bin/udev-block-notify
+    install -D -m0644 systemd/udev-block-notify.service $out/lib/systemd/user/udev-block-notify.service
+
+    install -D -m0644 README.md $out/share/doc/udev-block-notify/README.md
+    install -D -m0644 README.html $out/share/doc/udev-block-notify/README.html
+    install -D -m0644 screenshots/usb.png $out/share/doc/udev-block-notify/screenshots/usb.png
+    install -D -m0644 screenshots/optical.png $out/share/doc/udev-block-notify/screenshots/optical.png
+
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://github.com/eworm-de/udev-block-notify";
+    description = "Notify about udev block events";
+    mainProgram = "udev-block-notify";
+    license = lib.licenses.gpl3Only;
```

```suggestion
    license = lib.licenses.gpl3Plus;
```

https://github.com/eworm-de/udev-block-notify/tree/main?tab=readme-ov-file#license-and-warranty